ldap2pg/ldap.py

```diff
@@ -210,10 +210,7 @@ def connect(**kw):
         conn = UnicodeModeLDAPObject(conn)
 
     conn = LDAPLogger(conn)
-
-    if not options.get('REFERRALS'):
-        logger.debug("HOTFIX https://stackoverflow.com/questions/18793040/python-ldap-not-able-to-bind-successfully")
-        conn.set_option(ldap.OPT_REFERRALS, 0)
+    conn.set_option(ldap.OPT_REFERRALS, options.get('REFERRALS', False))
```

Comment:
This seems like a good place to leave a comment about why we touch this setting at all. (If you like that sort of thing in comments.)

The unclear error message would be good to include.

Reply:
You're right !